cardano-config/src/Cardano/Config/Shelley/OCert.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Cardano.Config.Shelley.OCert
+  ( OperationalCertError(..)
+  , readOperationalCert
+  , shelleyGenOperationalCertificate
+  , writeOperationalCert
+  ) where
+
+import           Cardano.Prelude
+
+import qualified Cardano.Binary as CBOR
+import           Control.Monad.Trans.Except.Extra
+                   (firstExceptT, handleIOExceptT, hoistEither)
+import qualified Data.ByteString.Lazy.Char8 as LB
+
+import           Shelley.Spec.Ledger.Keys (SKey, VKeyES, sign)
+import           Shelley.Spec.Ledger.OCert
+import           Shelley.Spec.Ledger.Serialization
+import           Ouroboros.Consensus.Shelley.Protocol.Crypto (TPraosStandardCrypto)
+
+
+shelleyGenOperationalCertificate
+  :: VKeyES TPraosStandardCrypto
+  -> SKey TPraosStandardCrypto
+  -- | Counter.
+  -> Natural
+  -- | Start of key evolving signature period.
+  -> KESPeriod
+  -> OCert TPraosStandardCrypto
+shelleyGenOperationalCertificate hotKESVerKey signingKey counter kesPeriod' = do
```

Comment:
Name bikeshedding:
* We don't need "shelley" in the name here since it's implicit from the module name.
* This isn't generate really but sign.

How about `signOperationalCertificate`?